hedera-node/src/main/java/com/hedera/services/ledger/HederaLedger.java

```diff
@@ -480,7 +476,8 @@ public void customize(AccountID id, HederaAccountCustomizer customizer) {
 	/**
 	 * Updates the provided {@link AccountID} with the {@link HederaAccountCustomizer}. All properties from the
 	 * customizer are applied to the {@link MerkleAccount} provisionally
-	 * @param id target account
+	 *
+	 * @param id         target account
```

Comment:
nit: whitespace necessary?

Reply:
It's an automatic formating config in my IDE for java docs